cypress/e2e/consortia/inventory/search-in-inventory/title-look-up-plugin-in-requests-app-use-shared-facet-in-member.cy.js

```diff
@@ -0,0 +1,343 @@
+import Permissions from '../../../../support/dictionary/permissions';
+import Affiliations, { tenantNames } from '../../../../support/dictionary/affiliations';
+import Users from '../../../../support/fragments/users/users';
+import TopMenu from '../../../../support/fragments/topMenu';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import ConsortiumManager from '../../../../support/fragments/settings/consortium-manager/consortium-manager';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import DataImport from '../../../../support/fragments/data_import/dataImport';
+import UserEdit from '../../../../support/fragments/users/userEdit';
+import ServicePoints from '../../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Requests from '../../../../support/fragments/requests/requests';
+import NewRequest from '../../../../support/fragments/requests/newRequest';
+import TitleLevelRequests from '../../../../support/fragments/settings/circulation/titleLevelRequests';
+import SelectInstanceModal from '../../../../support/fragments/requests/selectInstanceModal';
+
+function createFOLIOAndUploadMARCInstanceViaApi(titlesFOLIOInstance, marcFile, createdRecordsIds) {
+  titlesFOLIOInstance.forEach((title) => {
+    InventoryInstance.createInstanceViaApi({
+      instanceTitle: title,
+    }).then((instanceData) => {
+      createdRecordsIds.push(instanceData.instanceData.instanceId);
+    });
+  });
+
+  DataImport.uploadFileViaApi(marcFile.marc, marcFile.fileName, marcFile.jobProfileToRun).then(
+    (response) => {
+      response.entries.forEach((record) => {
+        createdRecordsIds.push(record.relatedInstanceInfo.idList[0]);
+      });
+    },
+  );
+}
+
+describe('Inventory', () => {
+  describe('Search in Inventory', () => {
+    const users = {};
+    const createdRecordsFromCentralTenant = [];
+    const createdRecordsFromUniversity = [];
+    const createdRecordsFromCollege = [];
+    const sharedFOLIOInstances = [
+      'C410702 Test Shared facet Instance 1 Shared Folio',
```

Comment:
It would be better to add a random postfix to instance names.

Reply:
Added. 
Thanks!